broker/src/main/java/io/zeebe/broker/exporter/stream/ExporterContainer.java

```diff
@@ -114,6 +115,11 @@ public void scheduleTask(final Duration delay, final Runnable task) {
     actor.runDelayed(delay, task);
   }
 
+  @Override
+  public ScheduledTask scheduleCancellableTask(final Duration delay, final Runnable task) {
+    return () -> actor.runDelayed(delay, task);
```

Comment:
This means that on `cancel()` it would execute `actor.runDelayed(delay, task)`. If not cancelled the task will never get executed. It should be as follows:
```
    final var delayedTask = actor.runDelayed(delay, task);
    return () -> delayedTask.cancel();
```

Reply:
Done